gems/pending/VMwareWebService/MiqPbmInventory.rb

```diff
@@ -0,0 +1,32 @@
+require 'VMwareWebService/PbmService'
+
+module MiqPbmInventory
+  def pbm_initialize(server, username, password)
+    @pbm_svc = PbmService.new(server, username, password) if @apiVersion >= '5.5'
```

Comment:
The PBM service piggy-backs on the VIM connection so the connection doesn't need to be explicitly closed.

However right now we're opening a RbVmomi::VIM connection to pass to RbVmomi::PBM which we should close, but I want to see if we can extract what we need (specifically the vc session cookie) from our existing Handsoap Vim connection.